### DISCUSSION TITLE
Video not loading just black screen fix?

### DISCUSSION THREAD

**Discussion by Yougonnayeet** [PoomSmart/YTABConfig#121] [Q&A]
Is there anything to fix this black screen situation thing in the config? When you go in like couple videos in, there would be a black screen thing on the video, then loads the video by closing and playing the video. Also I realised YouTube did the server side ad injection thing, did that affect the YouTube modded ipa stuff?

**Comment by Khiemduy2003** [comment]
Im in this problem too

**Comment by amourification** [comment]
Consistently having the same issue. I was able to solve it preciously by disabling the Adblock feature in settings but I cant find it anymore.

**Reply by Yougonnayeet** [reply]
Do you mean in the uyou settings?

**Reply by amourification** [reply]
Yes

**Reply by Yougonnayeet** [reply]
Well have you noticed the YouTube serverside ads thing that affected the modded ipas of YouTube? Also what version are you using on it?

**Reply by amourification** [reply]
No, not yet, fortunately.

**Comment by Yougonnayeet** [comment]
So has anyone figured it out or something, it’s been few weeks now

**Comment by PoomSmart** [comment]
No such config would have prevented this blank space issue - it's YouTube X thing.
